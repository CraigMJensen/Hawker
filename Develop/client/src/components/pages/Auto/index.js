import React from 'react';

function Auto() {
  return (
    <div>
      <h2>Auto</h2>
    </div>
  );
}

export default Auto;
