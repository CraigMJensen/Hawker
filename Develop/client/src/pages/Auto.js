import React from 'react';

const Auto = () => {

}

export default Auto;