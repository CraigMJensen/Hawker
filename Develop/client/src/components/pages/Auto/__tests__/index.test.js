// Import dependencies
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Auto from '..';

// After each test, make sure there isnt any leftover memory that could give you false results
afterEach(cleanup);

// Declare the components you are testing
describe('Auto component', () => {
  // verify the component is rendering properly
  it('Auto component renders', () => {
    render(<Auto />);
  });

  // compare snapshot versions of the DOM node structure
  it('matches snapshot DOM node structure', () => {
    const { asFragment } = render(<Auto />);
    expect(asFragment()).toMatchSnapshot();
  });
});