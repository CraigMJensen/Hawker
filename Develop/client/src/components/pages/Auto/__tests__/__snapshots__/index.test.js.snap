// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auto component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    <div
      class="title"
    >
      Auto
    </div>
    <div
      class="itemContainer"
    >
      <div
        class="card"
      >
        <div
          class="card-body"
        >
          <a
            class="btn btn-primary"
            href="#"
            type="button"
          >
            Check it out!
          </a>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
